### ISSUE TITLE
Change Terminology from SSL to TLS

### ISSUE THREAD

**Issue by wthayer** [mozilla/pkipolicy#184]
Update the policy to the modern term for SSL

**Comment by BenWilson-Mozilla** [comment]
Agreed - there is one anchor that references SSL - #61-ssl >6.1 SSL that would also have to be changed.

**Comment by BenWilson-Mozilla** [comment]
Public discussion started here - https://groups.google.com/a/mozilla.org/g/dev-security-policy/c/vfTx7Nm-44Y/m/qU1dGYwTAQAJ
